rbx_xml/src/types/binary_string.rs

```diff
@@ -20,7 +20,9 @@ impl XmlType<[u8]> for BinaryStringType {
         value: &[u8],
     ) -> Result<(), EncodeError> {
         writer.write(XmlWriteEvent::start_element(Self::XML_TAG_NAME).attr("name", name))?;
-        writer.write(XmlWriteEvent::cdata(&base64::encode(value)))?;
+        if value.len() > 0 {
```

Comment:
```suggestion
        if !value.is_empty() {
```

I think there's a Clippy lint that talks about this, but alas, they're not as warnings on this repo yet, heh.